### DISCUSSION TITLE
Scrollable Frame Stay At Bottom?

### DISCUSSION THREAD

**Discussion by authkevin** [TomSchimansky/CustomTkinter#1956] [Q&A]
Hey guys im currently working on a project where I need to keep a scrollable frame viewing the most recent line packed (or always at the bottom of the frame). What is the best way to do this? It says "see" is not a valid method.

**Comment by Akascape** [comment]
@authkevin Try this: 
```python
scrollable_frame._parent_canvas.yview_moveto(0.0)
```
